libs/common/utilities/src/lib/stories/stories.helper.ts

```diff
@@ -2,6 +2,19 @@ import { nothing } from 'lit';
 import * as prettier from 'prettier/standalone';
 import * as prettierBabel from 'prettier/parser-babel';
 
+import { action } from '@storybook/addon-actions';
+
+// Logt het event in zowel in Storybook als in de developer console.
+// Logt het event in de developer console omdat in Storybook custom event properties soms niet gelogd worden.
+export const logStorybookEvent = <T extends Event>(eventName: string) => {
```

Comment:
We hebben in een openstaande PR een common-storybook module gemaakt, we zullen eerst die inmergen en dan dit verhuizen naar daar.
Door deze methode in de common-utilities te plaatsen komt storybook mee in de bundel wat niet de bedoeling is.

Reply:
Top, merci !

Reply:
Dus afhankelijk van: https://github.com/milieuinfo/uigov-web-components/pull/116

Reply:
De build lijkt hierdoor dus ook te falen: https://github.com/milieuinfo/uigov-web-components/pull/116